packages/sw-appcache-behavior/build.js

```diff
@@ -32,7 +32,7 @@ module.exports = () => {
           commonjs(),
         ],
       },
-      outputName: 'build/client-runtime.js',
+      buildPath: 'build/client-runtime.js',
```

Comment:
This library is messy in that there are two different bundled outputs, one for consumption by the SW and the other for the browser. I didn't know how to cleanly list the metadata in the `package.json`'s `main`, since it wasn't clean whether it should be the path to the SW or browser script. So I decided to leave out the `package.json` metadata entirely.

Longer term, maybe splitting up the pieces into two different modules, each with their own `package.json` would make the most sense? Or some other refactoring that's outside the scope of this PR.